tests/tt_metal/tt_metal/perf_microbenchmark/common/util.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include "tt_metal/impl/device/device.hpp"
 #include "tt_metal/host_api.hpp"
+#include "tt_metal/hostdevcommon/dprint_common.h"
```

Comment:
Btw @tt-rkim I think this should fix the failing pipeline: https://github.com/tenstorrent/tt-metal/actions/runs/11079938437/job/30789716653

Sorry for breaking it to begin with!